mailchimp_auth/mailchimp.py

```diff
@@ -51,13 +52,23 @@ def put_supporter(self, user):
             "merge_fields": {
                 "FNAME": user.first_name,
                 "LNAME": user.last_name,
+            },
+            "interests": {
+                self.INTEREST_ID: True
             }
         }
 
+        tag_payload = {
+            "tags": [
+                {"name": "Database Sign-up", "status": "active"}
```

Comment:
let's have the tag be also pulled from settings. i bet we would like to differentiate pensions from salary.